package-lock.json

```diff
@@ -1,8 +1,6038 @@
 {
```

Comment:
Please only include the lock file if you changed the `package.json`.

Reply:
I removed the changes from the package-lock.json.